frontend/src/metabase/query_builder/components/filters/modals/BulkFilterSelect/BulkFilterSelect.styled.tsx

```diff
@@ -18,7 +18,7 @@ const lightSelectButton = ({ hasValue, isActive }: SelectFilterButtonProps) => `
     }
     padding: 0.5rem 1rem;
 
-    background-color: ${hasValue ? alpha("brand", 0.2) : "transparent"};
+    background-color: ${hasValue ? alpha("brand", 0.2) : color("white")};
```

Comment:
we want inputs to have white backgrounds so they stand out when hover/highlighted